spec/views/posts/show.html.erb.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
```

Comment:
* When i run the tests i noticed that some tests did not show the results because the files were not named correctly for a test file you should add spec to your file naming ( ie index.html.erb_spec.rb)